llvm/test/Transforms/LowerWGLocalMemoryCall/group_local_memory.ll

```diff
@@ -0,0 +1,286 @@
+; RUN: opt -S -sycllowerwglocalmemorycall < %s | FileCheck --check-prefixes=CHECK,CHECK-CALL %s
```

Comment:
Please, reduce the test case by removing unnecessary instructions, metadata, etc.

BTW, why do we need `CHECK-CALL` prefix?

Reply:
It is used to check that there is no calls of __sycl_allocateLocalMemory left in IR after the pass.